seqr/utils/elasticsearch/es_search.py

```diff
@@ -1273,11 +1280,11 @@ def _get_compound_het_page(grouped_variants, start_index, end_index):
 
 
 def _parse_es_sort(sort, sort_config):
-    if sort in {'Infinity', '-Infinity'}:
+    if sort in {'Infinity', '-Infinity', None}:
         # ES returns these values for sort when a sort field is missing, using the correct value for the given direction
         sort = maxsize
     elif hasattr(sort_config, 'values') and any(cfg.get('order') == 'desc' for cfg in sort_config.values()):
-        sort = sort * -1
+        sort = float(sort) * -1
```

Comment:
It is confusing to me that `sort` can have a variety of types. If adding a brief description of the purpose of the function `_parse_es_sort()`, it will be helpful for understanding the changes of `sort`.

Reply:
elasticsearch suports sorting on any type of data, its just whatever is in the field you specify. All of our sorts should be numeric, as we only support on numeric fields or using custom sorts that return numbers. However it turns out some of our predictor sort are loaded as strings and not numbers (i.e. the string `"0.01"`) so we need to explicitly cast them